openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -13,12 +13,70 @@
 from django.dispatch import Signal
 from django.utils.http import cookie_date
 
+from edx_rest_framework_extensions.auth.jwt import cookies as jwt_cookies
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
 from student.models import CourseEnrollment
 
+from .waffle import JWT_COOKIES_FLAG
+
+
 CREATE_LOGON_COOKIE = Signal(providing_args=['user', 'response'])
 
 
+LOGGED_IN_COOKIE_NAMES = (
+    # Header and payload sections of a JSON Web Token containing user
+    # information and used as an access token.
+    jwt_cookies.jwt_cookie_header_payload_name(),
+
+    # Signature section of a JSON Web Token.
+    jwt_cookies.jwt_cookie_signature_name(),
+
+    # Refresh token, which can be used to get a new JSON Web Token.
+    jwt_cookies.jwt_refresh_cookie_name(),
+
+    # TODO (ARCH-245): Remove the following deprecated cookies.
+    # Set to 'true' if the user is logged in.
+    settings.EDXMKTG_LOGGED_IN_COOKIE_NAME,
+
+    # JSON-encoded dictionary with user information.
+    settings.EDXMKTG_USER_INFO_COOKIE_NAME,
+)
+
+
+def is_logged_in_cookie_set(request):
+    """Check whether the request has logged in cookies set. """
```

Comment:
Nit: Spacing different at start and end of comment.

Reply:
fixed.